packages/core/src/types.ts

```diff
@@ -107,7 +107,10 @@ export interface ActionMeta<
 // or just make machine a behavior
 export type Spawner = <T extends ActorBehavior<any, any> | string>( // TODO: read string from machine behavior keys
   behavior: T,
-  name?: string
+  options?: Partial<{
+    name: string;
```

Comment:
shouldn't this be named `id`? it's passed as `id` here: 
https://github.com/statelyai/xstate/blob/686565da4ff9084b2492c59300d6f34db0b9744c/packages/core/src/spawn.ts#L40